docs/data-ai/quickstart/capture.md

```diff
@@ -0,0 +1,9 @@
+---
+linkTitle: "Capture image data"
+title: "Quickstart: Capture image data"
+weight: 20
+layout: "docs"
+type: "docs"
+description: "Capture images on your machine to use for inference."
```

Comment:
Is that needed for a quickstart even? Would it not be better to use inference on live images? Then you cut out one additional step? You don't show data capture then - is that vital?